src/common/layout/Header.js

```diff
@@ -211,10 +220,12 @@ class Header extends React.Component<Props, State> {
     window.addEventListener('scroll', this.handleScroll, true);
   }
 
-  handleScroll = ({ target }: SyntheticInputEvent<HTMLInputElement>) => {
+  handleScroll = ({ target }: _SyntheticInputEvent) => {
```

Comment:
does it work with `currentTarget`? https://github.com/facebook/flow/issues/2050#issuecomment-326773333

Reply:
No :'( For example `scrollTopMax` is missing